R/sql-clause.R

```diff
@@ -53,10 +53,19 @@ sql_clause_select <- function(
     top <- as.integer(top)
   }
 
+  sql_distinct <-
+    if (isTRUE(distinct)) {
+      " DISTINCT"
```

Comment:
Our usual style would be to put `sql_distinct <- ` in each branch or extract out a helper function. I'd probably lean towards a helper function since you could inline that in `clause` below.